backend/app/api/v1/commons/telco.py

```diff
@@ -40,58 +45,71 @@ async def getData(
         "latest_time": "{}T23:59:59".format(end_datetime.strftime("%Y-%m-%d")),
         "output_mode": "json",
     }
-    searchList = " OR ".join(
+    test_type_filter = " OR ".join(
```

Comment:
I shouldn't be commenting on lines you didn't change, but in other places you've changed the awkward old `str.format()` substitutions to more readable and modern f-strings -- that'd look better here, too. 😁

Reply:
Modified it :)